### ISSUE TITLE
Deeplink not working, only opens new Mingo

### ISSUE THREAD

**Issue by Christilut** [mingo-app/mingo#313]
This is my current deeplink (hasn't changed for over a year):
```
mingo://rekordcloud/production/documents/users/123?useCurrentTab=false
```

It only opens a new Mingo without any tabs. If Mingo is already open, it opens another Mingo instance without tabs.

On Windows 10, Mingo 1.6.4

**Comment by tothradoslav** [comment]
yep, I know about this, but it's more tricky than I thought with the multi-window support. Working on it, will be solved!

**Comment by Christilut** [comment]
Maybe you can add a setting to disable multi window support? I don't use it anyway since there are tabs and having Mingo properly open from my own admin website was awesome, when it last worked a few years ago...

**Comment by Christilut** [comment]
Or just make it open a new tab in the last opened Mingo.. Or let it only work if there is exactly 0 or 1 Mingo's open, but not when there are more open

**Comment by tothradoslav** [comment]
This has been fixed in release candidate [v2025.11.0-rc1](https://github.com/mingo-app/mingo/releases/tag/v2025.11.0-rc1).
Please test and verify the fix. This will be included in the next production release.

If you encounter any issues with the fix or implementation, please reply to this issue.

**Comment by Christilut** [comment]
I just tried the nightly on mac for this, it seems to work much better but not quite perfect. When mingo is open, it works as expected. When mingo is not open, it does open mingo but not the collection (just shows dashboard or the last open collections).

**Comment by tothradoslav** [comment]
This has been fixed in release candidate [v2025.11.0-rc1](https://github.com/mingo-app/mingo/releases/tag/v2025.11.0-rc1).
Please test and verify the fix. This will be included in the next production release.

If you encounter any issues with the fix or implementation, please reply to this issue.

**Comment by tothradoslav** [comment]
I improved this.

**Comment by tothradoslav** [comment]
This has been fixed in release candidate [v2025.11.0-rc3](https://github.com/mingo-app/mingo/releases/tag/v2025.11.0-rc3).

Please test and verify the fix. This will be included in the next production release.

If you encounter any issues with the fix or implementation, please reply to this issue.

**Comment by tothradoslav** [comment]
This has been fixed and released in version [2025.12.0](https://github.com/mingo-app/mingo/releases/tag/v2025.12.0).

Thank you for reporting this issue!
